lib/beacon-verification.ts

```diff
@@ -110,9 +119,16 @@ function signatureBuffer(sig: string) {
 }
 
 function roundBuffer(round: number) {
-    const buffer = Buffer.alloc(8)
-    buffer.writeBigUInt64BE(BigInt(round))
-    return buffer
+    if (!Number.isInteger(round) || round < 0) {
```

Comment:
the check here makes sense, though I dunno the rationale behind replacing `writeBigUint64BE`